packages/ui/src/components/internal/ThemeSelector.svelte

```diff
@@ -1,28 +1,34 @@
 <script lang="ts">
   import { getContext } from 'svelte'
+  import { Themes } from '@anticrm/theme'
   import Mute from './icons/Mute.svelte'
-  import WiFi from './icons/WiFi.svelte'
 
-  const { setTheme } = getContext('theme')
+  let { current, setTheme } = getContext('theme')
+
+  function changeTheme () {
+    switch (current) {
+      case Themes.LIGHT:
+        current = Themes.GREY
+        break;
+      case Themes.GREY:
+        current = Themes.DARK
+        break;
+      case Themes.DARK:
+        current = Themes.LIGHT
+        break;
+    }
+    setTheme(current)
```

Comment:
This looks to be overkill :) why not `setTheme(themes[current % themes.length])` where `themes = ['light', 'dark',...]`

Reply:
Some overengineering, I think in future we need changing icons (when we will have them), and store user setting

Reply:
So? :)

Reply:
Ping @BykhovDenis